jest.config.js

```diff
@@ -18,6 +18,6 @@ module.exports = {
   ],
   roots: [process.cwd()],
   testMatch: [
-    '**/jest/**/*.test.js',
+    '**/jest/**/(*.)(spec|test).js?(x)',
```

Comment:
Do we need to support `.spec` tests?

Reply:
I copied this from our other mono repos. I honestly don't remember exactly how it works but know that we updated it to avoid trying to run jest on the .test.jsx devsite files (I think)

Reply:
i reverted this for now